bin/cli.js

```diff
@@ -158,6 +158,10 @@ if (opts.slow) {
   mocha.slow(opts.slow)
 }
 
+if (opts.require) {
+  require(fs.absolute(opts.require));
+}
+
```

Comment:
What happens if the user specifies `--require` twice? mocha allows for that. I'm not sure if the casper cli parser supports it.